introduction/what-is.md

```diff
@@ -0,0 +1,45 @@
+This file holds the text to accompany the [What is InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321608) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What is InnerSource?
+
+The basic idea of InnerSource is this: If a consuming team can't get what it needs by submitting a feature request then it may submit a pull request instead.
+The producing team partners to review and accept the pull request and the functionality it implements.
+In this context of a code submission, we call the consuming team the **Guest** team and the producing team the **Host** team.
+
+The terms **Guest** and **Host** suggest a situation analogous to receiving a visitor in the home.
```

Comment:
This is a very nice explanation of the _Host/Guest_ analogy. Kudos, @rrrutledge !